### ISSUE TITLE
distinctUntilChanged export error on build beta.37

### ISSUE THREAD

**Issue by IsaacSomething** [angular/flex-layout#1389]
# Bug Report

#### What is the current behavior?
Breaks build on npm start

#### What are the steps to reproduce?
Update to latest version `"@angular/flex-layout": "^13.0.0-beta.37"`

#### What is the use-case or motivation for changing an existing behavior?
So that project can build

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
`@angular/core": "^13.2.0"`
`"@angular/material": "^13.2.0"`
`"typescript": "~4.4.4"`
`"rxjs": "~6.6.7"`

#### Is there anything else we should know?

![error](https://user-images.githubusercontent.com/25283027/151651105-55fa7467-b58b-40f2-ba00-6c11430c0c1d.PNG)

I assume `distinctUntilChanged` needs to be imported from `'rxjs/operators'`

Downgrading to `beta.36` resolves the issue

**Comment by CaerusKaru** [comment]
This is because when we upgraded the repo to use the Angular CLI, we also upgraded our internal version of RxJS to v7, which exports `distinctUntilChanged` from the top-level export. Of course v6 does not, so this means that _unofficially_, we have dropped support for RxJS v6 in beta 37. #1390 will rectify this, and I'll try to cut a new minor version some time next week to resolve this. In the meantime, the workaround is to upgrade to RxJS v7, or, as you mentioned, downgrade to beta 36.
